test/WebSites/HtmlGenerationWebSite/Views/HtmlGeneration_Home/Link.cshtml

```diff
@@ -12,6 +12,9 @@
     <!-- Globbed link tag with existing file -->
     <link asp-href-include="**/site.css" rel="stylesheet" title="&lt;the title>" />
 
+    <!-- Globbed link tag with comma separated pattern -->
+    <link asp-href-include="~/*.css, ~/sub/*.css" rel="stylesheet" title="&lt;the title" />
```

Comment:
Well then, since it works, include it in a test :smiley_cat: -- preferably `GlobbingUrlBuilderTest` since the repetition is certainly not something we'd like to see in a mainline scenario.